webapp/channels/src/components/post_view/post_recent_reactions/post_recent_reactions.tsx

```diff
@@ -9,7 +9,7 @@ import Permissions from 'mattermost-redux/constants/permissions';
 import {getEmojiName} from 'mattermost-redux/utils/emoji_utils';
 
 import ChannelPermissionGate from 'components/permissions_gates/channel_permission_gate';
-import WithTooltip from 'components/with_tooltip';
+import WithTooltip from 'components/with_tooltip/with_tooltip_new';
```

Comment:
0/5 wonder if it is better to keep `with_tooltip` and move the old one to `with_tooltip_old`

Reply:
if we move the old one then PR will lots of diffs thats why i didnt move